packages/components/pie-textarea/src/textarea.scss

```diff
@@ -1 +1,22 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-textarea {
+  --textarea-height: 72px;
+
+  height: var(--textarea-height);
+
+  &[data-pie-size="large"] {
+    --textarea-height: 80px;
+  }
+
+  &[data-pie-size="small"] {
+    --textarea-height: 64px;
+  }
+
+  textarea {
+    width: 100%;
+    height: 100%;
+  }
+}
+
```

Comment:
I assume these values are just placeholders right?

Reply:
yes, these are temporary values, only for visual tests